kubeai/install.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Copyright (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+helm upgrade --install opea -n kubeai kubeai/kubeai \
+    --create-namespace \
+    --set secrets.huggingface.token=$HF_TOKEN \
+    -f $DIR/opea-values.yaml
```

Comment:
Personally, I like HF token being taken from a file (e.g. `hf-token.txt`), and given to scripts (untested):
```suggestion
if [ -n "$HF_TOKEN" ]; then
    echo "Using HF_TOKEN env var"
elif [ -n "$1" ] && [ -f "$1" ]; then
    echo "Using given HF token file"
    HF_TOKEN=$(cat $1)
else
    echo "Usage: [HF_TOKEN=<token>] ${0##*/} [hf-token file]"
    echo "ERROR: HF token missing!"
    exit 1
fi

helm upgrade --install opea -n kubeai kubeai/kubeai \
    --create-namespace \
    --set secrets.huggingface.token="$HF_TOKEN" \
    -f "$DIR/opea-values.yaml"
```

Reply:
fixed.